static/less/panel.less

```diff
@@ -206,6 +206,13 @@
     }
 }
 
+.panel-info{
+    border:2px solid #5bc0de
+}
+.panel-info .panel-heading{
+    color:#5bc0de
```

Comment:
Please do not use hardcoded color values. Instead use color variables like `color:@info` 

You can find available color variables here:
https://github.com/humhub/humhub/blob/master/static/less/variables.less#L12

Reply:
Thanks! Done